src/Data/Units/PartsPer.purs

```diff
@@ -0,0 +1,26 @@
+-- | A collection of parts-per-x units.
+module Data.Units.PartsPer where
```

Comment:
Minor suggestion: maybe call this `PartsPerX`?

Reply:
Done.